### ISSUE TITLE
torch run with pth bad result 

### ISSUE THREAD

**Issue by lix19937** [lix19937/dnn#14]
```py
torch.save(model_object, 'model.pth')


model = torch.load('model.pth')
```

```
torch.save(model_object.state_dict(), 'params.pth')

model_object=model()
model_object.load_state_dict(torch.load('params.pth'))
model.eval()  
```

**Comment by lix19937** [comment]
Batch Normalization   

Dropout

**Comment by lix19937** [comment]
Models are now set in evaluation mode by default when instantiated with the from_pretrained() method. 
To train them don’t forget to set them back in training mode (model.train()) to activate the dropout modules.

**Comment by lix19937** [comment]
Models are now set in evaluation mode by default when instantiated with the from_pretrained() method. 
To train them don’t forget to set them back in training mode (model.train()) to activate the dropout modules.

**Comment by lix19937** [comment]
在多块GPU上进行训练会使得每一层网络名称多一个 **module.** 的后缀，

只在CPU上训练则没有 **module.** 这个后缀

```py
new_state_dict = OrderedDict()
for k in ckpt_of_OrderedDict:
    name = k.replace('module.', '')
    new_state_dict[name] = checkpoint['model_state_dict'].setdefault(k)
self.model.load_state_dict(new_state_dict)
```
